client-registry-api/src/test/java/uk/gov/di/authentication/clientregistry/services/ClientConfigValidationServiceTest.java

```diff
@@ -68,7 +70,8 @@ private static Stream<Arguments> registrationRequestParams() {
                         null,
                         null,
                         null,
-                        null),
+                        null,
+                        Channel.WEB.getValue()),
```

Comment:
We should also test `channel = STRATEGIC_APP` for one of these or a new argument